checkpoint_ngfw/help.md

```diff
@@ -27,26 +27,88 @@ The connection configuration accepts the following parameters:
 |port|integer|443|True|Check Point server port|None|443|
 |server|string|None|True|Check Point server IP address|None|198.168.2.1|
 |ssl_verify|boolean|True|True|Use SSL verification|None|True|
-|username_password|credential_username_password|None|True|Username and password|None|None|
+|username_password|credential_username_password|None|True|Username and password|None|{"username": "xxxxxx", "password": "xxxxxx"}|
 
 Example input:
 
 ```
 {
   "port": 443,
-  "server": "192.168.1.1",
-  "ssl_verify": false,
-  "username_password": {
-    "password": "xxxxxx",
-    "username": "xxxxxx"
-  }
+  "server": "198.168.2.1",
+  "ssl_verify": true,
+  "username_password": "{\"username\": \"xxxxxx\", \"password\": \"xxxxxx\"}"
```

Comment:
Fix this.

Reply:
It's autogenerated by icon-plugin, it's going to get overwritten every time :(

Reply:
that's what we have to do for accuracy

Reply:
it's not bad when you regen and do `git add -p` to selectively add changes